AppointEase/AppointEase.Application.Contracts/Models/DatabaseModels/TblAdmin.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace AppointEase.Application.Contracts.Models.DbModels;
+
+public partial class TblAdmin
```

Comment:
Why is a `DatabaseModel `inside the `ApplicationLayer `since we already have a layer for entities?